lib/src/base-styles/_nx-btn.scss

```diff
@@ -38,8 +38,8 @@
   }
 }
 
-.nx-btn, .nx-segmented-btn {
-  + .nx-btn {
+.nx-btn, .nx-segmented-btn, .nx-dropdown {
+  + .nx-btn, + .nx-segmented-btn, + .nx-dropdown {
```

Comment:
Should this more properly be `.nx-icon-dropdown` than `.nx-dropdown`.  Afterall other kinds of dropdowns don't have side margin.